src/reformatters/contrib/noaa/ndvi_cdr/ndvi_cdr/analysis/region_job.py

```diff
@@ -62,11 +60,14 @@ def out_loc(self) -> Mapping[Dim, CoordinateValueOrRange]:
 class NoaaNdviCdrAnalysisRegionJob(
     RegionJob[NoaaNdviCdrDataVar, NoaaNdviCdrAnalysisSourceFileCoord]
 ):
-    download_parallelism: int = 4
+    download_parallelism: int = 12
```

Comment:
might need to go down to 8 (which is what succeeded for me locally) or configure timeout stuff. Or maybe it'll be fine on k8s.